### PR TITLE
GH-43: connection info

### LINKED ISSUE #43: show connection info for videoplayer
Screenshot 2022-12-05 at 11 33 15

### PR DESCRIPTION
**Pull request by alex-waverley**
a user should be informed about connections to robot
Screenshot 2022-12-16 at 10 20 45

Closes #43

### CODE REVIEW

File: frontend/src/components/videoControls/components/Fullscreen/index.js [hunk 1]
```diff
@@ -4,23 +4,39 @@ import { Tooltip, SvgIcon, IconButton } from '@mui/material'
 import { useTheme } from '@mui/material/styles'
 import {ReactComponent as FullscreenIcon} from './fullscreen.svg'
 import {ReactComponent as FullscreenExitIcon} from './fullscreenexit.svg'
-import { useStore } from '../../../store'
+import { useStore } from '../../../../store'
 
 const Fullscreen = observer(() => {
-  const {videoPlayerStore: { isFullscreen, toggleFullscreen }} = useStore()
+  const {
+    videoPlayerStore: { isFullscreen, toggleFullscreen},
```
Comment (sskorol) [inline frontend/src/components/videoControls/components/Fullscreen/index.js:11]: bad formatting

File: frontend/src/components/videoControls/components/Fullscreen/index.js [hunk 2]
```diff
@@ -4,23 +4,39 @@ import { Tooltip, SvgIcon, IconButton } from '@mui/material'
...
+    rosStore: { isWSConnected, isTeleopReady },
+  } = useStore()
 
   const fullscreenIcon = () => {
     return isFullscreen ? FullscreenExitIcon : FullscreenIcon
   }
 
   const theme = useTheme()
 
+  const connected = isWSConnected && isTeleopReady
```
Comment (sskorol) [inline frontend/src/components/videoControls/components/Fullscreen/index.js:21]: It seems like these two constants are duplicated in several components. Try to optimize it.

**Review by sskorol — CHANGES_REQUESTED** [review]

**Comment by sskorol** [comment]
@alex-waverley seems like the screenshot wasn't uploaded for some reason.

File: frontend/src/components/videoControls/components/Fullscreen/index.js (hunk 1 above)
Comment (alex-waverley) [reply frontend/src/components/videoControls/components/Fullscreen/index.js]: done

File: frontend/src/components/videoControls/components/Fullscreen/index.js (hunk 2 above)
Comment (alex-waverley) [reply frontend/src/components/videoControls/components/Fullscreen/index.js]: optimized for components related to CameraControls

**Review by sskorol — APPROVED** [review]

**Pull request merged**